prober/tcp.go

```diff
@@ -107,6 +111,14 @@ var (
 				expect: "OK",
 			},
 		},
+		"postgresql": []queryResponse{
```

Comment:
```suggestion
		"postgres": []queryResponse{
```

One small nitpick, could we change the name of this option to `postgres` to match [the same option in `openssl`](https://www.openssl.org/docs/man1.1.1/man1/openssl-s_client.html):
> Currently, the only supported keywords are "smtp", "pop3", "imap", "ftp", "xmpp", "xmpp-server", "irc", __"postgres"__, "mysql", "lmtp", "nntp", "sieve" and "ldap".

Reply:
Yes, sure `postgres` is fine for me, the reason why I chose `postgresql` is that in connection URI it is `postgresql`: https://www.postgresql.org/docs/13/libpq-connect.html#LIBPQ-CONNSTRING

Reply:
Updated.